### PR TITLE
Fix ufl compatibility issues for `perpendicular-flap/solid-fenics` and `elastic-tube-3d/solid-fenics`

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
See #504.

Replacing `nabla_div` and `nabla_grad` with `div` and `grad`.

### CODE REVIEW

**Review by MakisH — COMMENTED** [review]
For the perpendicular-flap, this seems to do the trick and produce the same results:

tutorials-perpendicular-flap-displacement-all-watchpoints

I ported the same changes in the elastic-tube-3d. This takes too long to run till the end, but running till t=0.0042 seems to already produce reasonable results. At least the result looks symmetric and I can see the pulse propagating. I will eventually let it run completely and update the images in https://precice.org/tutorials-elastic-tube-3d.html#post-processing.

I also removed some left-over mentions to ufl in the form of `if ufl: <A> else: <B>`, since the first would now be a dead branch and confusing.

We still use ufl in the `solid-dunefem` case (I still want to run that).

**Review by MakisH — APPROVED** [review]
The following cases work for me:

- `elastic-tube-3d/solid-fenics`
- `perpendicular-flap/solid-fenics`
- `flow-over-heated-plate/solid-dunefem` (which still uses ufl, but there seems to be no dependency issue)

File: elastic-tube-3d/solid-fenics/solid.py [hunk 1]
```diff
@@ -140,8 +131,8 @@ def update_fields(u, u_old, v_old, a_old):
     v0_vec, a0_vec = v_old.vector(), a_old.vector()
 
     # call update functions
-    a_vec = update_a(u_vec, u0_vec, v0_vec, a0_vec, ufl=False)
-    v_vec = update_v(a_vec, u0_vec, v0_vec, a0_vec, ufl=False)
+    a_vec = update_a(u_vec, u0_vec, v0_vec, a0_vec)
+    v_vec = update_v(a_vec, u0_vec, v0_vec, a0_vec)
```
Comment (BenjaminRodenberg) [inline elastic-tube-3d/solid-fenics/solid.py:135]: The `if `ufl`: ... else: ...` is not a dead branch. See below.

File: elastic-tube-3d/solid-fenics/solid.py [hunk 2]
```diff
@@ -152,8 +143,8 @@ def avg(x_old, x_new, alpha):
     return alpha * x_old + (1 - alpha) * x_new
 
 
-a_np1 = update_a(du, u_n, v_n, a_n, ufl=True)
-v_np1 = update_v(a_np1, u_n, v_n, a_n, ufl=True)
+a_np1 = update_a(du, u_n, v_n, a_n)
+v_np1 = update_v(a_np1, u_n, v_n, a_n)
```
Comment (BenjaminRodenberg) [inline elastic-tube-3d/solid-fenics/solid.py:147]: Here `ufl=True`

**Review by BenjaminRodenberg — COMMENTED** [review]
I don't think that removing the `ufl` part is a good idea. It's also explicitly done this way in the tutorial that this case is based on. See https://comet-fenics.readthedocs.io/en/latest/demo/elastodynamics/demo_elastodynamics.py.html. @MakisH Can you please revert this part?

**Pull request merged**